### PR TITLE
OASYS - Edit contract information and assignment

### PR DESCRIPTION
**Pull request by hamidnesar**
These are old requirements, hence there are no Jira Story Ids available for them.

### CODE REVIEW

File: src/test/java/CUSTOM_BUSINESS/Oasys/Features/Contracts.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Feature: Contract Flow
+
+Background: common steps
+Given User is logged in the application and navigated to Contract Administration
+
+@ContractInformation @NESARH2 @Regression @playwright
```
Comment (matakevin) [inline src/test/java/CUSTOM_BUSINESS/Oasys/Features/Contracts.feature:6]: Is there a ticket ID you can tag or is this independent?

**Review by matakevin — APPROVED** [review]
Very clean code, LGTM

**Review by puzikovam2 — APPROVED** [review]
approved by @puzikovam2

**Review by sarwarahmed1 — APPROVED** [review]

**Review by alenaminsk — APPROVED** [review]

File: src/test/java/CUSTOM_BUSINESS/Oasys/Features/Contracts.feature (hunk 1 above)
Comment (hamidnesar) [reply src/test/java/CUSTOM_BUSINESS/Oasys/Features/Contracts.feature:6]: These are old requirements that doesn’t exist on Jira.

**Pull request merged**